gluecodium/src/main/java/com/here/gluecodium/generator/common/modelbuilder/AbstractLimeBasedModelBuilder.kt

```diff
@@ -55,13 +57,19 @@ abstract class AbstractLimeBasedModelBuilder<E>(
     /**
      * Store a result which has a direct mapping to a lime model name. This is used to transform
      * links in comments from lime to language model. "Ambiguous" LIME references (i.e. when the
-     * disambiguation suffix is omitted) are also allowed.
+     * disambiguation suffix is omitted) and full-signature function references are also allowed.
      */
     protected fun storeNamedResult(limeElement: LimeNamedElement, element: E) {
         storeResult(element)
         referenceMap[limeElement.fullName] = element
         val ambiguousKey = limeElement.path.withSuffix("").toString()
         referenceMap[ambiguousKey] = element
+        if (limeElement is LimeFunction) {
+            val signatureKey = ambiguousKey + limeElement.parameters
+                .joinToString(prefix = "(", postfix = ")") { it.typeRef.toString() }
```

Comment:
I think it would be better using `separator = ","` and stripping whitespace for lookup to avoid being sensitive to sane formatting of multiple parameters.

Reply:
Done